analysis/topeft_run2/analysis_processor.py

```diff
@@ -1019,10 +1180,13 @@ def process(self, events):
 
                             # Loop over the appropriate AR and SR for this channel
                             for appl in cat_dict[nlep_cat][njet_val]["appl_lst"]:
-
+                                weight_tmp = weight
                                 # We don't want or need to fill SR histos with the FF variations
                                 if appl.startswith("isSR") and wgt_fluct in data_syst_lst: continue
 
+                                # We only want the photon fakerate weight if we are considering the AR with explicit photon requirement
+                                if appl == "isAR_2lOS_ph": weight_tmp = weight_tmp * events.fakerate_ph_val
```

Comment:
Just to be safe we can warp this in a `if self.ttA_analysis:` block.

Reply:
I will also fix this.

Reply:
Fixed now.